### PR TITLE
Error handling for tweets

### LINKED ISSUE #961: Error Handling for Tweets
- The webapp provide error in console if loklak server is down
- Add error handling and show the text "No Tweets Available" instead of blank area in view.
errortwiter

### PR DESCRIPTION
**Pull request by Heredroky**
Fixes #961 

Hi, here you can see a preview: https://obscure-escarpment-32909.herokuapp.com/

When Loklak doesn't get any tweet, and when the webpage catches that error, it tells to the user that there are no available tweets.
Here are some photos
screen shot 2017-01-07 at 9 58 28 pm
screen shot 2017-01-07 at 9 58 12 pm

I hope you like,
Cheers! 😄

### CODE REVIEW

File: src/backend/assets/dependencies/tweets.js [hunk 1]
```diff
@@ -76,33 +76,37 @@ function timeSince(date) {
...
+				}
 			} else {
-				parsed += "<a href='" + words[word] + "' target='_blank' style='word-break:break-all'>" + words[word] + "</a> ";
+				parsed += words[word] + " ";
 			}
-		} else {
-			parsed += words[word] + " ";
 		}
+		document.getElementById("tweet").innerHTML = parsed;
+		var user = data.statuses[tweetNum].user;
```
Comment (Heredroky) [inline src/backend/assets/dependencies/tweets.js]: User is totally used

**Comment by Princu7** [comment]
@Heredorky Great work. One issue. In almost everywhere in the codebase, we have used two spaces instead of four for indentation. Could you please do it that way, please!

**Comment by Abhi2424shek** [comment]
You got the name wrong it's @Heredroky @Princu7

File: src/backend/assets/dependencies/tweets.js [hunk 2]
```diff
@@ -49,60 +49,64 @@ function lastTweet() {
...
+		var tweetDate = new Date(data.statuses[tweetNum].created_at);
+		document.getElementById("dateTweeted").innerHTML = "Tweeted " + timeSince(tweetDate) + " ago";
```
Comment (aayusharora) [inline src/backend/assets/dependencies/tweets.js]: ## Suggestion:
@Heredroky Please save all the getElementById in global function closure, `var  dataTweeted = document.getElementById("dateTweeted")` and use name instead accessing the DOM again and again.

File: src/backend/assets/dependencies/tweets.js [hunk 3]
```diff
@@ -49,60 +49,64 @@ function lastTweet() {
...
-    interval = Math.floor(seconds / 3600);
-    if (interval > 1) {
-        return interval + " hours";
-    }
-    interval = Math.floor(seconds / 60);
-    if (interval > 1) {
-        return interval + " minutes";
-    }
-    return Math.floor(seconds) + " seconds";
+	if (interval > 1) {
```
Comment (aayusharora) [inline src/backend/assets/dependencies/tweets.js]: @Heredroky remove extra spaces !!

**Comment by aayusharora** [comment]
@Heredroky Thanks! Great work, there are some requirements. Please see the comments.

File: src/backend/assets/dependencies/tweets.js (hunk 2 above)
Comment (Heredroky) [inline src/backend/assets/dependencies/tweets.js]: @aayusharora Sorry, I didn't understand well 😞 , Where do I have to save the variables?

**Comment by Heredroky** [comment]
@Princu7 Check the changes, I fix the indentation issue
@aayusharora Check it out, I create the variables and placed them on the top of the page

Can it be merged?

**Comment by aayusharora** [comment]
@Heredroky can you squash the commits please?

**Comment by Heredroky** [comment]
@aayusharora It is squashed now 😄

**Comment by Heredroky** [comment]
@aayusharora Can it be merged?

**Comment by aayusharora** [comment]
Thanks @Heredroky . Merging this :+1:

**Pull request merged**